apps/src/studio/studio.js

```diff
@@ -153,6 +154,16 @@ function loadLevel() {
   // protagonistSpriteIndex was originally mispelled. accept either spelling.
   Studio.protagonistSpriteIndex = level.protagonistSpriteIndex || level.protaganistSpriteIndex;
 
+  switch (level.customGameType) {
+    case 'Big Game':
+      Studio.customGame = new BigGameInfo();
```

Comment:
Does AudioPlayer do anything functional, or is it mostly just there for clarity?